recipes/sixs_json/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export FFLAGS=$(echo "${FFLAGS}" | sed "s/-fopenmp//g")
+
+cmake ${CMAKE_ARGS} -D CMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR
+
+make -j${CPU_COUNT}
+make install
```

Comment:
Is there a `make check`  or `ctest` we can run here?

Reply:
I've modified the CMakeLists created by Pete Bunting to support 6sV2.1
I don't see any test in this file, but I can look into creating some.

Reply:
While having tests is a good thing, if they are not there now we as a third party packager, won't require you to write them for this PR. Please add the tests later when they are available.